test/Integration.t.sol

```diff
@@ -182,6 +215,10 @@ contract GeneralisedTests is CapAutomatorIntegrationTestsBase {
             uint256 postDecreaseSupplyCap = pool.getReserveData(assets[i]).configuration.getSupplyCap();
 
             assertEq(postDecreaseSupplyCap, currentSupply + newGap);
+
+            ( ,,, uint48 lastUpdateBlock, uint48 lastIncreaseTime ) = capAutomator.supplyCapConfigs(assets[i]);
+            assertEq(lastUpdateBlock,  block.number);
+            assertEq(lastIncreaseTime, 0);
```

Comment:
Same here. These should be asserted to be different values before `exec` is called.